wire-schema/src/main/java/com/squareup/wire/schema/Location.java

```diff
@@ -26,23 +25,15 @@
 @AutoValue
 public abstract class Location {
   public static Location get(String path) {
-    return get("", path);
-  }
-
-  public static Location get(String base, String path) {
-    base = CharMatcher.is('/').trimTrailingFrom(base);
     path = CharMatcher.is('/').trimLeadingFrom(path);
-    return new AutoValue_Location(base, path, -1, -1);
+    return new AutoValue_Location(path, -1, -1);
   }
 
   public Location at(int line, int column) {
-    return new AutoValue_Location(base(), path(), line, column);
+    return new AutoValue_Location(path(), line, column);
   }
 
-  /** Returns the base of this location; typically a directory or .jar file. */
-  public abstract String base();
```

Comment:
It can stay, there just has to be some hackery in the tests which use the real filesystem and now require copying a subset of protos to a separate folder for processing.

I basically have to do `content.replace(giantTemoraryFolderString, "../wire-runtime/things/are/pretty");` to make it work. The current test setup is sorely in need of overhaul.